packages/ui/components/ui/tx/view/value/index.tsx

```diff
@@ -28,7 +28,7 @@ export const ValueViewComponent = ({
     const { amount = new Amount(), metadata } = view.valueView.value;
     const exponent = getDisplayDenomExponent.optional()(metadata);
     const formattedAmount = formatAmount(amount, exponent);
-    const symbol = metadata.symbol || 'Unknown Asset';
+    const symbol = metadata.symbol || metadata.display;
```

Comment:
We mistakenly displayed assets with undefined symbol as “Unknown assets”
In fact, these assets are known, and this was causing confusion. 

All  IBC assets should have a `symbol` field defined, it is required in cosmos registry 
https://github.com/cosmos/chain-registry/blob/62b3fa019a0db5a277925859999ab430420a3fa6/assetlist.schema.json#L36

But some penumbra internal assets (such as lpnft) don't have this field, so we have to use `display`